.lando.yml

```diff
@@ -0,0 +1,75 @@
+name: simplytest-me
+recipe: drupal9
+
+config:
+  php: "7.3"
+  webroot: web
+  xdebug: false
+  database: mariadb:10.3
+
+events:
+  # Clear cache after a database import
+  post-db-import:
+    - appserver: cd $LANDO_WEBROOT && drush cache-rebuild --yes
+
+services:
+  appserver:
+    run:
+      - cd $LANDO_MOUNT && composer install --no-scripts
+
+  nodejs:
+    type: node:14
+    globals:
+      gulp-cli: latest
```

Comment:
I don't believe Gulp is used anymore, it's an artifact gulpfile. LaravelMix as a webpack wrapper is use.

Reply:
I will test removing it.